prompt.go

```diff
@@ -70,10 +78,16 @@ func (p *Prompt) Run() {
 
 				// Unset raw mode
 				// Reset to Blocking mode because returned EAGAIN when still set non-blocking mode.
+				p.inNotTearDown = false
 				debug.AssertNoError(p.in.TearDown())
 				p.executor(e.input)
 
 				p.completion.Update(*p.buf.Document())
```

Comment:
It looks you can call `exitor` before this line.

Reply:
@c-bata that might work. I will update the PR with this. No need for `inNotTearDown`  in that case.